helsingborg-stad/aterbruket-webapp
# Feature/button in header when scrolled & remove reservation

**Explain the changes you’ve made**
Made button in header show when big button is scrolled by and disappears.
Made the add available again when clicking the "ta bort reservation" text

**Explain why these changes are made**
user friendly to see a button in header when the big button disappears from the screen. makes is easy to haffa/hämta ut wherever you are on the screen.
Remove reservation: because you should be able to change your made and un-reserv it
 
**Explain your solution**
Use addEventListener and removeEventListener on scroll, put href to the button to make it the element and see its position on the screen, and when that disappears a small btn in the header is showing
 
**How to test the changes?**
To test the button show in header-function:
Go to detail page, scroll up and down to see a button in header show when big button disappears. 
Click "haffa-knapp" and see scroll up and down to see "hämta ut"-knapp in header show when big btn disappears in "reserverad-vy"

To test the remove reservation-function:
Click text "ta bort reservation" and the add will go back to "tillgänglig"
